### PR TITLE
Update ADOPTERS.md w/ Expedia Group

### PR DESCRIPTION
**Pull request by guillermomanzo**
#### :heavy_check_mark: Checklist

- [ ] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ x] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)

### CODE REVIEW

File: ADOPTERS.md [hunk 1]
```diff
@@ -17,3 +17,4 @@
 | [Fiverr](https://www.fiverr.com)             | [@nirga](https://github.com/nirga)                           | Unifying separate tools that developers are using today (i.e. monitoring, dead letter queues management, etc.) into a single platform. |
 | [Zalando SE](https://www.zalando.de)         | [@leviferreira](https://github.com/leviferreira)             | Building V2 of the Internal Development Portal.                                                                                        |
 | [LegalZoom](https://legalzoom.com)           | [@backjo](https://github.com/backjo)                         | Developer portal - hub for all engineering projects and metadata.                                                                      |
+| [Expedia Group](https://www.expediagroup.com)| [Mike Turner](miturner@expediagroup.com), [Sneha Kumar](snkumar@expediagroup.com), [@guillermomanzo](https://github.com/guillermomanzo)         | EG Common Developer Toolkit
```
Comment (adamdmharvey) [inline ADOPTERS.md:20]: Welcome aboard!!!!! 🎉 

The e-mails should lead with `mailto:` otherwise the links will end up as broken.

Comment (guillermomanzo) [reply ADOPTERS.md:20]: will fix

**Comment by Rugvip** [comment]
Sorry, hijacked to make this process a lil bit smoother, spell checking is no longer applied to `ADPTERS.md` 😅 

Also formatted the file, so should pass checks now

**Comment by Rugvip** [comment]
Oh, nevermind, workflow doesn't use the same script so updated the vocab instead ._.

**Review by Rugvip — APPROVED** [review]

**Pull request merged**